### PR TITLE
Task for deploying to Azure Automation

### PR DESCRIPTION
**Pull request by t-suson**
Tests folder will be modified

### CODE REVIEW

File: Tasks/AzureAutomationV0/DeployToAzureAutomation.ps1 [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    [string][Parameter(Mandatory=$false)]$ModuleStorageAccountName
+)
+
+$ErrorActionPreference = "Stop"
+$VerbosePreference = "Continue"
+
+# If the user wants to deploy runbooks to Azure Automation
+if ($AutomationRunbook -or ($RunbookFile -and (($RunbookFile.Split('.')[-1] -match "ps1") -or ($RunbookFile.Split('.')[-1] -match "py")))) {
+    if ($AutomationRunbook -and (-not($RunbookFile) -or (-not($RunbookFile.Split('.')[-1] -match "ps1") -or (-not($RunbookFile.Split('.')[-1] -match "py"))))) {
+        Write-Host "We are in the runbook if if "
```
Comment (KTOmega) [inline Tasks/AzureAutomationV0/DeployToAzureAutomation.ps1]: remove this line

File: Tasks/AzureAutomationV0/ImportAzureAutomationModules.ps1 [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+$ModuleZips = Get-ChildItem -Path $CompressedModulesPath -Filter *.zip -File
+
+ foreach ($ModuleZip in $ModuleZips)
+ {
+     # Save each module zip in a new blob inside the modules container
+    $StorageLocation = Set-AzureStorageBlobContent -Container 'modules' -Context $StorageAccount.Context -Blob $ModuleZip.Name -File $ModuleZip.FullName -Force
+    $Link = New-AzureStorageBlobSASToken -Container 'modules' -Blob $StorageLocation.Name -Context $StorageAccount.Context -FullUri `
+            -Permission rwd -StartTime (Get-Date) -ExpiryTime (Get-Date).AddMinutes(60)
+
+    Write-Host "Saved module $ModuleZip.Name to Storage Account container 'modules'... publishing to Azure Automation"
```
Comment (KTOmega) [inline Tasks/AzureAutomationV0/ImportAzureAutomationModules.ps1]: This should be `$($ModuleZip.Name)`

Comment (KTOmega) [reply Tasks/AzureAutomationV0/ImportAzureAutomationModules.ps1]: Same for below occurrence

**Comment by kmkumaran** [comment]
Closing this PR as the task is planned as Marketplace extension.

**Pull request closed without merging**